modules/ROOT/pages/create-outbound-message-flow.adoc

```diff
@@ -1,126 +1,139 @@
 
+[[reference-ids]]
+== Configure Reference IDs for the Partner and Host
+
+Partner Manager uses the partner reference ID and host reference ID to locate the appropriate message flow for routing outbound messages. For information about about how this routing process works, see xref:partner-manager-identifiers[Sender and Receiver Identifiers] and xref:outbound-message-routing.adoc[Outbound Message Routing].
+
+To configure a partner reference ID, either select *All available identifiers* or select a specific partner identifier.
+
+To configure a host reference ID, either select *Ignore* or select your organization's internal reference ID (such as the business unit name) as it appears in the data element of the source JSON or XML message from your backend systems.
+
+[[translation-map]]
 == Import the Translation Map
 
-=== Prerequisites
-To create your outbound EDI-to-application message (JSON or XML) map in Anypoint Studio, see xref:partner-manager-maps.adoc[EDI Translation Maps].
+A translation map in an outbound message flow transforms outbound messages from your organization's enterprise application format to the format your partners expect. Create a translation map by using DataWeave. For more information, see see xref:partner-manager-maps.adoc[EDI Translation Maps].
 
-=== Steps
+To import the translation map:
 
 . In the message flow, expand the *Map* section.
-. Click *Import* and then click *Choose file*.
-. In the *New mapping file* window, navigate to the DataWeave map you want to use and then click *Open*.
-. Click *Upload mapping*. +
+. Click *Import < Choose file*.
+. Navigate to the DataWeave map you want to use and click *Open*.
+. Click *Upload mapping*.
+
 The translation map is added to the message flow.
 
+[[send-to-partner-message-type]]
 == Configure the Target Message Type
 
+The target message type in an outbound message flow defines the structure of the messages that Partner Manager sends to the partner, either directly or via a 3rd party connection. Partner Manager supports the AS2, FTP, and SFTP formats for send to message types.
+
+For the EDIFACT and X12 message types, configuring the target message type also involves configuring acknowledgements (optional), creating custom EDIFACT settings (optional), and selecting EDIFACT identifiers to use for routing the outbound messages.
+
+To configure the target message type:
+
 . Expand the *Message type* section and click *Select*.
-. In the *Select message type* window:
-* To use an existing message type, select the message type from the list and click *Select*.
-* To create a new message type, click *New* next to the search box, define the xref:partner-manager-create-message-type.adoc#create-message-type[EDIFACT and X12 message type], and click *Save*.
+. In the *Select message type* window, do either of the following:
+* Select an existing send to message type and click *Select*.
```

Comment:
ditto re. "send to"
I do think we need some text in this deliverable to explain UI terminology vs. actual conceptual terminology in narrative.

Reply:
GLOBAL for "send to"